src/components/InputGroup/index.test.tsx

```diff
@@ -21,19 +21,19 @@ describe('InputGroup component', () => {
 
   describe('getFlexSettings', () => {
     it('sets expected default for 2 items', () => {
-      expect(getFlexSettings(2)).toStrictEqual(['grow', 'shrink']);
+      expect(getFlexSettings(2, [])).toStrictEqual(['grow', 'shrink']);
     });
 
     it('sets expected default for 3 items', () => {
-      expect(getFlexSettings(3)).toStrictEqual(['shrink', 'grow', 'shrink']);
+      expect(getFlexSettings(3, [])).toStrictEqual(['shrink', 'grow', 'shrink']);
     });
 
     it('sets expected default for 4 items', () => {
-      expect(getFlexSettings(4)).toStrictEqual(['shrink', 'grow', 'grow', 'shrink']);
+      expect(getFlexSettings(4, [])).toStrictEqual(['shrink', 'grow', 'grow', 'shrink']);
```

Comment:
Is there a change that would make it not necessary to repeatedly pass an empty array to this function? Whenever I see anything like this, it's a smell for something else that probably needs to be changed.

Reply:
we can change the 2nd param to be optional